### PR TITLE
CR/QA Leaderboard: implement it in React

### PR DESCRIPTION
**Pull request by danielbeardsley**
Even looks a tiny bit nicer.

### CODE REVIEW

**Comment by danielbeardsley** [comment]
QA :+1: 
image

**Review by bhuminson — APPROVED** [review]
CR :dagger: lgtm

**Comment by danielbeardsley** [comment]
RE QA :+1:  webpack builds are still successful.

**Review by bhuminson — APPROVED** [review]
CR 🗡️

**Review by andyg0808 — APPROVED** [review]

**Comment by andyg0808** [comment]
CR :goat: through a98e817

File: frontend/src/leader-list.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+export function LeaderList({leaders, title}: {leaders: SigCount[], title: string}) {
+   return (
+   <Box
+      px={6}
+      py={2}
+      fontSize="1rem"
+      bgColor="var(--leaderboard-background)">
+      <HStack spacing={2} textAlign="center">
+         <chakra.span p={1}>{title}:</chakra.span>
+         {leaders.map(({user, count}, i) =>
```
[the remark this answers is not in the document]
Comment (danielbeardsley) [reply frontend/src/leader-list.tsx]: Good call.

**Comment by danielbeardsley** [comment]
QA :+1: 

Still stands 

image

**Comment by danielbeardsley** [comment]
Merging cause the last commit was itty bitty and was just following a CR suggestion.

**Pull request merged**